Automattic/newspack-plugin
# Whitelist Plugin: Fix Missing Media

### All Submissions:


### Changes proposed in this Pull Request:

Whitelists the Fix Missing Media plugin, which is typically used on staging sites before launch: https://github.com/Automattic/fix-missing-media

### How to test the changes in this Pull Request:

1. If Fix Missing Media is installed, de-activate and delete it.
1. Check out `add/fix-missing-media-plugin`.
1. Navigate to Plugins admin page.
1. Install Fix Missing Media.
1. Navigate to Newspack->Health Check and verify it is not listed there.

### Other information:
